src/components/CDPList.js

```diff
@@ -1,37 +1,81 @@
-import React, { memo, Fragment, useEffect, useState } from 'react';
+import debounce from 'lodash.debounce';
+import React, { memo, useEffect, useState, useRef, useMemo } from 'react';
 import styled from 'styled-components';
-// import { ReactComponent as MakerSmall } from '../images/maker-small.svg';
 import { ReactComponent as Plus } from '../images/plus.svg';
-import { Flex, Text } from '@makerdao/ui-components-core';
+import { ReactComponent as NavUp } from '../images/nav-up-icon.svg';
+import { ReactComponent as NavDown } from '../images/nav-down-icon.svg';
+import { Flex, Text, Box } from '@makerdao/ui-components-core';
 import RatioDisplay from './RatioDisplay';
-import { Link } from 'react-navi';
+import { Link, useCurrentRoute } from 'react-navi';
 import useModal from 'hooks/useModal';
 import useMaker from 'hooks/useMaker';
 import useStore from 'hooks/useStore';
 import { trackCdpById } from 'reducers/multicall/cdps';
 import { getCdp, getCollateralizationRatio } from 'reducers/cdps';
 import round from 'lodash/round';
 import { Routes } from '../utils/constants';
+import { useSpring, animated } from 'react-spring';
```

Comment:
are there still plans to use spring here?

Reply:
Yeah, I started using it initially, but I was having trouble getting it to work properly. I wanted to revisit it again when the rest of the reqs are finished to open & close the cdp list.